onnxruntime/core/framework/sequential_executor.cc

```diff
@@ -68,17 +68,17 @@ static void CalculateTotalOutputSizes(OpKernelContextInternal* op_kernel_context
 #if defined(TRACE_EXECUTION)
       const TensorShape& tensor_shape = tensor.Shape();
       std::cout << node_name << " output[" << i << "]"
-                         << " size=" << tensor_size
-                         << " shape=" << tensor_shape.ToString()
-                         << " element_size=" << tensor.DataType()->Size()
-                         << "\n";
+                << " size=" << tensor_size
+                << " shape=" << tensor_shape.ToString()
+                << " element_size=" << tensor.DataType()->Size()
+                << "\n";
 #endif
       total_output_sizes += tensor_size;
     }
   }
 }
 
-static void CalculateTotalInputSizes(const OpKernelContextInternal* op_kernel_context,
+void SequentialExecutor::CalculateTotalInputSizes(const OpKernelContextInternal* op_kernel_context,
```

Comment:
Instead of changing these, can you just copy the implementation to partial_executor.cc and keep them static here?

(It would be disproportionately nicer if we could avoid changing this file at all in this PR. We can then do a follow-on PR to merge as much as possible of `PartialExecutor` and `SequentialExecutor` in a way that should be obviously uncontroversial.)